### PR TITLE
created a profile for @Grumpyyash as a contributor and community member

### LINKED ISSUE #1704: [Community] Member Profile: Yash Vardhan
**Current Behavior**

@Grumpyyash has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @Grumpyyash  as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: Grumpyyash
- Twitter: 
- LinkedIn: yash-vardhan-688952188
- Link to profile picture:

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by buquio**
Signed-off-by: buquio <kehinde.bukola14@gmail.com>

**Description**

This PR fixes #1704

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [✔ ] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/members/yash-vardhan-grumpyyash/index.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: Yash Vardhan Grumpyyash
```
Comment (Tanuj22) [inline src/collections/members/yash-vardhan-grumpyyash/index.mdx:2]: name: Yash Vardhan

File: src/collections/members/yash-vardhan-grumpyyash/index.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (Tanuj22) [inline src/collections/members/yash-vardhan-grumpyyash/index.mdx:1]: Keep the folder name as `yash-vardhan`

File: src/collections/members/yash-vardhan-grumpyyash/index.mdx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+name: Yash Vardhan Grumpyyash
+position: Contributor
+image_path: https://drive.google.com/file/d/1wkpIPkjEbJyxBj3ayl_nkG00DRlXEgSd/view?usp=sharing
```
Comment (Tanuj22) [inline src/collections/members/yash-vardhan-grumpyyash/index.mdx:4]: image_path: ./Yashvardhan.jpg

File: src/collections/members/yash-vardhan-grumpyyash/index.mdx [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+github: Grumpyyash
+twitter: Yash_dn
+linkedin: yash-vardhan-688952188
+bio:  Undergraduate at IIT Dhanbad (India). A Developer, Programmer and a Keen Learner
+blurb: Mesh Head
+status: Active
+published: true
+---
+
```
Comment (Tanuj22) [inline src/collections/members/yash-vardhan-grumpyyash/index.mdx:13]: Remove this extra line

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Thank you for this @buquio . Please do not push any changes in `package-lock.json`

File: src/collections/members/yash-vardhan-grumpyyash/index.mdx (hunk 1 above)
Comment (buquio) [reply src/collections/members/yash-vardhan-grumpyyash/index.mdx:2]: ok will apply changes

File: src/collections/members/yash-vardhan-grumpyyash/index.mdx (hunk 2 above)
Comment (buquio) [reply src/collections/members/yash-vardhan-grumpyyash/index.mdx:1]: ok

Comment (Tanuj22) [reply src/collections/members/yash-vardhan-grumpyyash/index.mdx:1]: This folder can now be deleted

**Comment by leecalcote** [comment]
@buquio thanks for creating this profile. I went ahead and made a couple updates to help tidy a couple small items. @Tanuj22 and @Grumpyyash, we're ready for a review here.

**Comment by leecalcote** [comment]
Good work here, @buquio 👍

**Comment by l5io** [comment]
🚀 Preview for commit 41f19b45121e544767f734add7867476a25bf5e1 at: https://60669760b3f8a4aeae71f49a--layer5ng.netlify.app

**Review by Grumpyyash — CHANGES_REQUESTED** [review]
Good work @buquio and thank you!

Just one thing, do we really need to change package-lock.json for this @leecalcote? I think we should revert back if we are not installing any new package.

Rest, the profile looks good :) 
Thank you @buquio , @leecalcote and @Tanuj22 sir!

**Comment by leecalcote** [comment]
You're right, @Grumpyyash. I don't know that there's a need to update the package-lock.json. Given that this bump is inconsequential (given that the site builds and functions well), I don't know that its worth the hassle of reverting.

Pretty cool, @Grumpyyash - it's neat thing to see your profile up here! 🎈 

Excellent work, @buquio 😄

File: src/collections/members/yash-vardhan-grumpyyash/index.mdx (hunk 1 above)
Comment (leecalcote) [reply src/collections/members/yash-vardhan-grumpyyash/index.mdx]: Perhaps, Yash (Grumpy) Vardhan is more appropriate....     😉

**Review by leecalcote — APPROVED** [review]

**Pull request merged**